rotkehlchen/exchanges/utils.py

```diff
@@ -32,3 +47,59 @@ def deserialize_asset_movement_address(
             value = None
 
     return value
+
+
+def create_binance_symbols_to_pair(
+    exchange_data: Dict[str, Any],
+    location: Location,
+) -> Dict[str, BinancePair]:
+    """Parses the result of 'exchangeInfo' endpoint and creates the symbols_to_pair mapping
+    """
+    result: Dict[str, BinancePair] = {}
+    for symbol in exchange_data['symbols']:
+        symbol_str = symbol['symbol']
+        if isinstance(symbol_str, FVal):
+            # the to_int here may rase but should never due to the if check above
```

Comment:
```suggestion
            # the to_int here may raise but should never due to the if check above
```

Reply:
Lol the dictionary didn't catch this :(

Reply:
That's because it's American English for "raze".  To destroy something completely